src/js/components/DataTable/Body.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import { TableRow, TableCell } from '../Table';
+import { InfiniteScroll } from '../InfiniteScroll';
+
+import Cell from './Cell';
+import { StyledDataTableBody, StyledDataTableRow } from './StyledDataTable';
+
+const Body = ({ columns, data, primaryProperty, size, theme, ...rest }) => (
+  <StyledDataTableBody size={size} theme={theme} {...rest}>
+    <InfiniteScroll
+      items={data}
+      scrollableAncestor='window'
+      renderMarker={marker => (
+        <TableRow><TableCell>{marker}</TableCell></TableRow>
+      )}
+    >
+      {datum => (
+        <StyledDataTableRow key={datum[primaryProperty]} size={size}>
+          {columns.map(column => (
+            <Cell
+              key={column.property}
+              context='body'
```

Comment:
what is the difference between `context` and `scope` ? The variable name seems very similar.

Reply:
Ah, OK. "context" means header vs. body vs. footer. "scope" is an accessibility term for tables. So, I think we should keep "scope" but perhaps rename "context".

Reply:
👍

Reply:
After looking into it, `Table` uses the a React context value in exactly the same way. So, I agree that this is a bit overloaded with the React term, but I think it still makes the most sense as a term. I'm open to alternative suggestions though.